Package.swift

```diff
@@ -26,14 +29,14 @@ let package = Package(
                  url: "https://github.com/Sage-Bionetworks/JsonModel-Swift.git",
                  from: "1.4.9"),
         .package(name: "AssessmentModel",
-                 url: "https://github.com/Sage-Bionetworks/AssessmentModelKMM.git",
-                 from: "0.8.3"),
+                 url: "../AssessmentModelKMM", from: "0.8.4"),
+//                 url: "https://github.com/Sage-Bionetworks/AssessmentModelKMM.git",
+//                 from: "0.8.4"),
```

Comment:
You'll want to switch this back.

Reply:
I switched it back, and also changed the AssessmentModelKMM version from 0.8.4 to 0.8.6.